wsf_scraping/items.py

```diff
@@ -16,6 +16,7 @@ class WHOArticle(BaseArticle):
     year = scrapy.Field()
     types = scrapy.Field()
     subjects = scrapy.Field()
+    authors = scrapy.Field()
```

Comment:
we only expect authors in WHO cases right? when did this disappear from the code :p

Reply:
It magically disapeared when I introduced a `BaseArticle` item 😇